portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,41 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```

Comment:
What do you think of importing the specific library you want to use instead?

At Google, we don't use the wildcard because it's difficult to tell what libraries your code is using if you want/need to refactor. However, today I learned that using the wildcard has no performance impact!(https://stackoverflow.com/questions/7128348/performance-difference-between-a-wild-card-import-and-the-required-class-import)

Reply:
Smart, I'll do that, thanks!